mlflow/server/js/src/components/CompareRunScatter.js

```diff
@@ -100,43 +103,57 @@ class CompareRunScatter extends Component {
             </div>
           </form>
           <div className="col-xs-9">
-            <ResponsiveContainer width="100%" aspect={1.55}>
-              <ScatterChart>
-                <XAxis type="number" dataKey='x' name='x'>
-                  {this.renderAxisLabel('x')}
-                </XAxis>
-                <YAxis type="number" dataKey='y' name='y'>
-                  {this.renderAxisLabel('y')}
-                </YAxis>
-                <CartesianGrid/>
-                <Tooltip
-                  isAnimationActive={false}
-                  cursor={{strokeDasharray: '3 3'}}
-                  content={this.renderTooltip}
-                />
-                <Scatter
-                  data={scatterData}
-                  fill='#AE76A6'
-                  isAnimationActive={false}
-                />
-              </ScatterChart>
-            </ResponsiveContainer>
+            <Plot
+              data={[
+                {
+                  x: xs,
+                  y: ys,
+                  text: tooltips,
+                  hoverinfo: "text",
+                  type: 'scatter',
+                  mode: 'markers',
+                  marker: {
+                    size: 10,
+                    color: "rgba(200, 50, 100, .75)"
+                  },
+                },
+              ]}
+              layout={{
+                margin: {
+                  l: 40,
+                  r: 40,
+                  b: 30,
+                  t: 30
+                },
+                hovermode: "closest",
+                xaxis: {
+                  title: this.encodeHtml(this.state["x"].key)
+                },
+                yaxis: {
+                  title: this.encodeHtml(this.state["y"].key)
+                }
+              }}
+              className={"scatter-plotly"}
+              config={{
+                responsive: true,
```

Comment:
It could be cool to allow zooming with the mousewheel here by adding `scrollZoom: true` (see [plotly docs](https://plot.ly/javascript/configuration-options/#scroll-and-zoom)). However adding that doesn't seem to work for me, although it does seem ([link](https://community.plot.ly/t/scroll-and-zoom-on-react/15485/2)) to have worked for others.

Reply:
It looks like this is not working due to a bug in Plotly: https://github.com/plotly/plotly.js/issues/2371 so we can wait for that to be fixed.